src/game-support/bioshock-infinite.md

```diff
@@ -0,0 +1,11 @@
+# Bioshock Infinite (Steam)
+<!-- script:Aliases [
```

Comment:
These aliases are not required, please delete and change to `<!-- script: Aliases [] -->`